docs/accessibility/accessibility-statement.md

```diff
@@ -13,15 +13,23 @@ eleventyNavigation:
 
 ## Overview
 
-The Office of Technology Services, and Pelican, Louisiana’s Design System, are committed to providing digital products that are accessible. We are actively working to increase the accessibility and usability of our digital products. This commitment to accessibility begins with this site.
+Accessibility is a continual part of the development process. It has to be implemented along with content development, UX, wireframe, design, and development choices. 
+
+The Office of Technology Services (O.T.S.), and Pelican, Louisiana’s Design System, are committed to providing digital products that are accessible. We are actively working to increase the accessibility and usability of our digital products.
+
+## Your Role in Accessibility
+
+O.T.S. works to create a culture which cares about accessibility. Part of caring means developing the skills to help create accessible products. O.T.S. has also created <a href="https://accessibility.ots.la.gov" target="_blank">Accessible Web Services</a> to help create a caring culture. This resource presents our roles towards creating accessible products. There are also numerous resource links to help grow your skills for accessibility.
```

Comment:
This is the markdown way to do a target _blank. Additionally do we want to make this live before the a11y site is live?

```suggestion
O.T.S. works to create a culture which cares about accessibility. Part of caring means developing the skills to help create accessible products. O.T.S. has also created [Accessible Web Services](https://accessibility.ots.la.gov){target="_blank"} to help create a caring culture. This resource presents our roles towards creating accessible products. There are also numerous resource links to help grow your skills for accessibility.
```